roles/edpm_nova/scripts/derive_pci_passthrough_whitelist.py

```diff
@@ -0,0 +1,587 @@
+#!/usr/bin/python3
+
+#
+# Copyright 2019 Red Hat Inc.
```

Comment:
We can keep this here, if this is an unmodified copy of an existing script. Otherwise it must be updated to 2024